torch/utils/benchmark/utils/historic/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Fallback for back-testing earlier versions of PyTorch.
+cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
+
+project(timer_cpp_jit)
+set(CMAKE_CXX_STANDARD 14)
+
+message("name:            ${NAME}")
+message("torch_root:      ${TORCH_ROOT}")
+message("is_standalone:   ${IS_STANDALONE}")
+message("link_libtorch:   ${LINK_LIBTORCH}")
+message("valgrind_include ${VALGRIND_INCLUDE}")
+
+if(${IS_STANDALONE})
+    add_executable(${NAME} "${PROJECT_SOURCE_DIR}/${NAME}.cpp")
+else()
+    find_package(Python3 COMPONENTS Interpreter Development REQUIRED)
+    find_package(PythonInterp REQUIRED)  # Without this PyBind11 will find the wrong version
+    find_package(pybind11 CONFIG REQUIRED)
+    add_compile_definitions(TORCH_EXTENSION_NAME=${NAME})
+    pybind11_add_module(${NAME} "${PROJECT_SOURCE_DIR}/${NAME}.cpp")
+endif()
+
+# Wall time doesn't need this, but it doesn't hurt.
+target_include_directories(${NAME} PRIVATE ${VALGRIND_INCLUDE})
```

Comment:
This line is a bit surprising for two reasons: (1) at least whenever I've used it, Valgrind was dunked in the standard /usr/include so I didn't actually need to do anything to include the header, and (2) even if it wasn't, typical operating procedure in cmake files is to use `find_package` to find the appropriate includes. Curious why you needed it like this here.

Reply:
Answer: this is pointing to the valgrind wrapper directory, not the system valgrind installation.